.CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contribute to the Polkadot Developer Docs
+
+TODO
```

Comment:
IRT these TODOs - will we use issues to track them, or just have a couple PRs which takes care of various markdown files like this?  I assume Papermoon has some boilerplate stuff for this as well?

Reply:
Yes, this is on my to-do list. I plan to add a couple PRs to address these. We have some existing boilerplate content, but I'm revisiting it to add clarity to certain sections, especially since we’ll have more external contributors involved.